_data/overviews.yml

```diff
@@ -89,6 +89,10 @@
         The Scala REPL is a tool (<code>scala</code>) for evaluating expressions in Scala.
         <br><br>
         The <code>scala</code> command will execute a source script by wrapping it in a template and then compiling and executing the resulting program
+    - title: JDK Compatibility
+      icon: coffee
+      url: "jdk-compatibility/overview.html"
+      description: "Tables and more explaining Scala's use of the Java Virtual Machine and Java Development Kit."
```

Comment:
The title and description are considerably broader than the actual content.

What do you think about:

```
title: JDK Version Compatibility
description: "Which Scala versions work on what JDK versions"
```

If the content were later to become broader, the description could be broadened as well.

Reply:
👍